readme.md

```diff
@@ -73,26 +116,60 @@ The default export is `remarkImages`.
 
 ### `unified().use(remarkImages)`
 
-Add an improved image syntax.
-Transform URLs in text that reference images (`png`, `svg`, `jpg`, `jpeg`, or
-`gif`) to images.
+Plugin to add a simpler image syntax.
+There are no options.
+
+## Syntax
 
-Supported URLs / URIs:
+This plugin looks for URLs and paths, on their own, that end in an image
+extension (`png`, `svg`, `jpg`, `jpeg`, or `gif`).
```

Comment:
maybe not a part of this PR, but it may make sense to add webp and avif to the defaults, or to add an option to customize supported extensions, maybe both

Reply:
potential follow up to this in https://github.com/remarkjs/remark-images/pull/5#issue-1040307437